TEST.md

```diff
@@ -397,3 +397,13 @@ Anything in <https://wg21.link/index.yaml> are linked automatically.
   - LWG Issues (e.g., `[@LWG1234]` [@LWG1234])
   - Github Edits (e.g, `[@EDIT1234]` [@EDIT1234])
   - Standing Documents (e.g., `[@SD6]` [@SD6])
+
+# Automatic Cross-References # {#auto-cross-ref}
+
+Automatic cross-references to a header can be achieved by the following syntax:
+
+You see 'Automatic Cross-Reference' text if you syntax above.
+
+Example:
+
+  - `[](#auto-cross-ref)` → [](#auto-cross-ref)
```

Comment:
The reason you were getting the red highlighting here is because inline code in the framework is highlighted as `cpp` by default. I guess the `` `[foo]{.cls}` `` above gets highlighted as `cpp` nicely enough, but `` `[foo](#id)` `` isn't working, so in this case we need to do `` `[foo](#id)`{.markdown} ``
```suggestion
Automatic header links are written as `[](#auto-header-links)`{.markdown}, and renders as [](#auto-header-links).
```

Reply:
Done